jvcproj/plugin.yaml

```diff
@@ -0,0 +1,25 @@
+# Metadata for the Smart-Plugin
+plugin:
+    # Global plugin attributes
+    type: interface                 # plugin type (gateway, interface, protocol, system, web)
+    description:
+        de: 'Plugin um JVC D-ILA Projektoren zu kontrollieren und durch jvcprojectortools erstellte Gammakurven zu uebertragen'
+        en: 'plugin to control JVC D-ILA projectors and transfer gammatables generated with jvcprojectortools'
+    maintainer: NPortz
+    tester: 
```

Comment:
Do you now any other testers to list here?

Reply:
No, sorry. Of course i know some People with that kind of projector. But most of them use Crestron to control the projector. I`m the only one who tested it at the moment.